addons/web/static/src/core/effects/rainbow_man.scss

```diff
@@ -1,256 +1,158 @@
-      display: inline-block;
+    .o_reward_star_01, .o_reward_star_03 {
+        animation: reward-stars $-reward-base-time ease-in-out infinite alternate-reverse;
+    }
 
-      // Reset margins for first and penultimate childs (the last one is shadow)
-      *:first-child {
-        margin-top: 0;
-      }
+    .o_reward_star_02, .o_reward_star_04 {
+        animation: reward-stars $-reward-base-time * 1.2 ease-in-out infinite alternate;
     }
 
-    .o_reward_msg_content {
-      position: relative;
-      font-family: sans-serif;
-      text-align: left;
-      color: $reward-text-color;
+    .o_reward_thumbup {
+        animation: reward-scale $-reward-base-time * 0.5 ease-in-out 0s infinite alternate;
     }
 
     .o_reward_shadow_container {
-      transform: translateY(0px) rotateZ(0);
-      animation: reward-float $reward-base-time ease-in-out $reward-base-time infinite alternate;
+        animation: reward-float $-reward-base-time ease-in-out infinite alternate;
     }
 
     .o_reward_shadow {
-      @include size(100%, 12px);
-      background-color: #e7eaf0;
-      border-radius: 100%;
-      transform: scale(0.8) rotateZ(0);
-      animation: reward-scale $reward-base-time ease-in-out $reward-base-time infinite alternate;
-      @include o-position-absolute(auto, auto, -40px, 0);
-    }
-  }
+        animation: reward-scale $-reward-base-time ease-in-out infinite alternate;
+    }
+
+    .o_reward_msg_container {
+        aspect-ratio: 1 / 1;
+        transform: translateY(5px);
+        animation: reward-float $-reward-base-time ease-in-out $-reward-base-time infinite alternate-reverse;
+    }
 }
 
 @keyframes reward-fading {
-  0% {
-    opacity: 0;
-  }
-  100% {
-    opacity: 1;
-  }
+    0% {
+        opacity: 0;
+    }
 }
 
 @keyframes reward-fading-reverse {
-  100% {
-    opacity: 0;
-  }
+    100% {
+        opacity: 0;
+    }
 }
 
 @keyframes reward-jump {
-  0% {
-    transform: scale(0.5);
-  }
-  50% {
-    transform: scale(1.05);
-  }
-  to {
-    transform: scale(1);
-  }
+    0% {
+        transform: scale(0.5);
+    }
+    50% {
+        transform: scale(1.05);
+    }
+    to {
+        transform: scale(1);
+    }
 }
 
 @keyframes reward-jump-reverse {
-  0% {
-    transform: scale(1);
-  }
-  50% {
-    transform: scale(1.05);
-  }
-  to {
-    transform: scale(0.5);
-  }
+    0% {
+        transform: scale(1);
+    }
+    50% {
+        transform: scale(1.05);
+    }
+    to {
+        transform: scale(0.5);
+    }
 }
 
 @keyframes reward-rainbow {
-  0% {
-    stroke-dashoffset: -500;
-  }
-  to {
-    stroke-dashoffset: 0;
-  }
+    to {
+        stroke-dashoffset: 0;
+    }
 }
 
 @keyframes reward-rainbow-reverse {
-  to {
-    stroke-dashoffset: -500;
-  }
+    from {
+        stroke-dashoffset: 0;
+    }
+    to {
+        stroke-dashoffset: -600;
+    }
 }
 
 @keyframes reward-float {
-  from {
-    transform: translateY(0px);
-  }
-  to {
-    transform: translateY(5px);
-  }
+    from {
```

Comment:
I think you can remove `from { }` and define the last animation step only.
Could you check the other animations too?

> If a 0% or from keyframe is not specified, then the user agent constructs a 0% keyframe using the computed values of the properties being animated. If a 100% or to keyframe is not specified, then the user agent constructs a 100% keyframe using the computed values of the properties being animated.
https://www.w3.org/TR/css-animations-1/#keyframes

Reply:
@stefanorigano 👋 
It's done 👍